charts/ztka/templates/configmap-fluentbit.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.deploy.filebeat.enable }}
```

Comment:
@nlamirault filebeat is used to collect audit logs into the system via elasticsearch whereas Fluentbit is used to collect audit logs into the system via a database. Both cannot be used together, why are we trying to control fluentbit's configuration if filebeat is enabled ?

Can you elaborate on the problem and what are we trying to achieve here ?

Reply:
it was a typo. fixed.
If we set `deploy.filebeat.enable: false`, this configmap is always deployed.